github-issue/in.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import requests
+import json
+import sys
+import os
+
+def _in(instream):
+    payload = json.load(instream)
+    source = payload["source"]
+
+    token = source["token"]
+    repository = source["repository"]
+    endpoint = "https://api.github.com/repos/" + repository + "/issues/" + number
+    headers = {'Authorization': 'token ' + token}
+
+    connection = requests.get(endpoint, headers=headers)
+    i = json.loads(connection.text)
+
+    for item in issueNumbers:
```

Comment:
this `in` script should only be passed one number/timestamp combo as the input version